alarm_trap_sender.cpp

```diff
@@ -157,7 +193,14 @@ void AlarmTrapSender::issue_alarm(const std::string& issuer, const std::string&
     {
       if (!AlarmFilter::get_instance().alarm_filtered(index, severity))
       {
+        alarmActiveTable_create_row((char*) "", AlarmTrapSender::alarm_time_issued(), alarm_counter, alarm_table_def);
```

Comment:
If you declare the first parameter of `alarmActiveTable_create_row` as a `const char*`, you don't need this cast. (Alternatively, as it's always the same, you could do away with it altogether.)